baseplate/file_watcher.py

```diff
@@ -63,15 +64,59 @@ class FileWatcher(object):
     :param callable parser: A callable that takes an open file object, parses
         or otherwise interprets the file, and returns whatever data is
         meaningful.
-    :param float timeout: How long, in seconds, to block instantiation waiting
```

Comment:
are you hard-wrapping at 80 columns? you can go up to 100!!!

Reply:
In my defense, this docstring was already doing that. 😅